src/content/en/2021/caching.md

```diff
@@ -17,4 +19,146 @@ featured_stat_3: TODO
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=885292598&format=interactive",
+  sheets_gid="2102749619",
+  sql_file="header_trends.sql"
+) }}
+
+Cache Control directives, 61% of mobile requests include a Cache-Control response header with a `max-age` directive.
+
+The most misused Cache Control directive is `set-cookie`, used for 0.07% of total directives for Desktop and 0.08% for Mobile.
+
+Fun fact the the largest max-age value is 1625540701883604800000 seconds or 51545557517871.8 years (51 trillion).
+
+{{ figure_markup(
+  caption="TODO",
+  content="51 trillion years",
+  classes="big-number",
+  sheets_gid="529870849",
+  sql_file="todo.sql"
+) }}
+
+Talk about `immutable` directive, 12% of desktop requests containing Cache-Control: immutable are on the "static.parastorage.com" host.
+
+Talk about invalid date, last modified and expires, (0.11% of dates in desktop requests are invalid).
+
+### Vary
+
+Talk about Vary adoption, 42% of requests on mobile set Vary: accept-encoding.
+
+The most common Vary headers are: `accept-encoding`, `user-agent`, `origin`, `accept`.
+
+### TTL adoption
+
+Talk about 28% of mobile requests use neither Cache-Control nor Expires headers.
+
+The median HTML resource on mobile has a TTL of 14 days
+
+## Cookie Cache adoption
+
+Talk about cookies.
+
+Fun fact `test_cookie` is the most popular desktop and mobile cookie name at 4% of all cookies.
+
+{{ figure_markup(
+  caption="TODO",
+  content="4%",
+  classes="big-number",
+  sheets_gid="455647224",
+  sql_file="todo.sql"
+) }}
+
+Top cookies attributes, SameSite is on 68% of cookies set in mobile responses.
+
+## What type of content are we caching?
+
+Talk about what kind of resources are we caching, (resource age groups: party, type).
+
+Talk about resource age, the median HTML resource on mobile is 9 weeks old.
+
+Talk about, total number of requests with comparable Last-Modified and expiration times, not necessarily all requests.
+
+Talk about 54% of mobile resources are older than their TTL.
+
+{{ figure_markup(
+  caption="TODO",
+  content="54%",
+  classes="big-number",
+  sheets_gid="768623684",
+  sql_file="todo.sql"
+) }}
+
+Total number of requests with comparable Last-Modified and expiration times, not necessarily all requests.
+
+Talk about, 60% of 1P mobile resources are older than their TTL, which is more common than 3P resources (45%).
+
+## How do cache TTLs compare to resource age?
+
+The median difference between resource TTL and resource age is -6 days, meaning the age is 6 days older than the TTL.
+
+## Identifying caching opportunities
+
+Talk about ~100% of fonts on mobile are cacheable!
+
+### Lighthouse TTL score
+Talk about 12% of mobile pages score 0.9 or higher on the long TTL audit
+
+### Lighthouse waysted bytes
+Talk about 59% of mobile pages could save 0 - 1 MB with caching
+
+## Future
+
+TODO ...
+
```

Comment:
From [Fonts](https://almanac.httparchive.org/en/2020/fonts#conclusion) chapter:[ Partitioning the cache](https://developers.google.com/web/updates/2020/10/http-cache-partitioning)